crates/extensions/c8y_mapper_ext/src/operations/upload.rs

```diff
@@ -79,7 +79,9 @@ impl OperationContext {
         command: &GenericCommandState,
     ) -> Result<(), ConversionError> {
         if command.is_finished()
-            && command.get_log_path().is_some()
+            && command
+                .get_log_path()
+                .is_some_and(|log_path| log_path.try_exists().is_ok())
```

Comment:
I would simply use [`exists`](https://docs.rs/camino/latest/camino/struct.Utf8PathBuf.html#method.exists) as `try_exists().is_ok()` returns true is a symlink is broken.

Reply:
I read the docs of `.exists()` too fast as those docs were recommending to use `try_exists()` instead...but I didn't go into the reasons too much.

Reply:
Fixed. Since this is a one-liner change, I didn't bother with a fix up commit and just force pushed the change directly.